fighters/pichu/src/acmd/aerials.rs

```diff
@@ -72,18 +83,18 @@ unsafe fn pichu_attack_air_b_game(fighter: &mut L2CAgentBase) {
     }
     frame(lua_state, 5.0);
     if is_excute(fighter) {
-        FT_ADD_DAMAGE(fighter, 2.0);
+        FT_ADD_DAMAGE(fighter, 2.0*(VarModule::get_float(boma.object(), vars::pichu::instance::CHARGE_RECOIL_MUL)));

```

Comment:
For all instances where you pull a value from VarModule, please don't nest it within another function for readability sake, instead do something like:
```rs
let recoil_mul = VarModule::get_float(boma.object(), vars::pichu::instance::CHARGE_RECOIL_MUL);
FT_ADD_DAMAGE(fighter, 2.0 * recoil_mul);
```